lib/rubocop/cop/mixin/array_min_size.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Common functionality for handling array count minimums
```

Comment:
That's a fair point. I've expanded the documentation on the mixin to specify which configuration option it is handling, and then expanded the doc comments on both `Style/SymbolArray` and `Style/WordArray` to explain the configuration parameter.